### PR TITLE
- build version updates

### PR DESCRIPTION
**Pull request by rakesh-parihar**
Updated SDK versions and gradle versions to cope with Android studio v3.2

### CODE REVIEW

**Review by k0shk0sh — CHANGES_REQUESTED** [review]
Please remove the build tools as they aren't needed anymore for a lib

File: permission/build.gradle [hunk 1]
```diff
@@ -2,12 +2,12 @@ apply plugin: 'com.android.library'
 apply from: 'maven-push.gradle'
 
 android {
-    compileSdkVersion 24
-    buildToolsVersion "24.0.2"
+    compileSdkVersion 28
+    buildToolsVersion '28.0.3'
```
Comment (k0shk0sh) [inline permission/build.gradle]: This can be removed

Comment (rakesh-parihar) [reply permission/build.gradle]: By removed you mean reset the version again to 24.X or completely remove the lines?

**Review by rakesh-parihar — COMMENTED** [review]
You may check now. I did not try altering info in of gradle file of a library.

**Comment by rakesh-parihar** [comment]
Removed all extra information from lib gradle.

**Comment by k0shk0sh** [comment]
By removing, I meant removing the line entirely as libs doesn't need builtools version defined anymore.

File: permission/build.gradle (hunk 1 above)
Comment (k0shk0sh) [reply permission/build.gradle]: Remove the line completely

**Comment by rakesh-parihar** [comment]
Could you guide me through the process of clearing this pull request? I've made some changes to my forked project but how they can be reflected here. Pardon for being childish.

**Comment by k0shk0sh** [comment]
@rakesh-parihar just commit & push your changes and they will be reflected here as well.

**Pull request closed without merging**